bench-vote/src/main.rs

```diff
@@ -199,8 +232,10 @@ fn main() -> Result<()> {
 
         if let Some(quic_params) = &quic_params {
             let quic_server_params = QuicServerParams {
-                max_connections_per_ipaddr_per_min: 1024,
-                max_connections_per_peer: 1024,
+                max_connections_per_ipaddr_per_min: 8192,
```

Comment:
Can someone get surprised if we change the value in a PR that is claiming to only add options?

Reply:
I added an option to control this as well. To allow higher connections count, this need to be increased  as well